BAZProject/BAZProjectTests/SearchViewModelTests.swift

```diff
@@ -0,0 +1,97 @@
+//
+//  SearchViewModelTests.swift
+//  BAZProject
+//
+//  Created by Manuel S on 20/04/23.
+//
+
+import XCTest
+@testable import BAZProject
+
+final class SearchViewModelTests: XCTestCase {
+    
+    var sut: SearchViewModel?
+    var sessionMock: SessionMock?
+
+    override func setUp(){
```

Comment:
Siempre es bueno dejar un espacio antes de las llaves. 
```suggestion
    override func setUp() {
```